docs/learning/ui/AugmentingUI.md

```diff
@@ -121,7 +93,7 @@ One last thing to point out in the above example. We specified default size for
 
 ## UiItemProviderOverrides
 
-When registering a [UiItemsProvider]($appui-abstract) with the [UiItemsManager]($appui-abstract) it is possible to pass an additional argument to limit when the provider is allowed to provide its items. The interface [UiItemProviderOverrides]($appui-abstract) defines the parameters that can be used to limit when the provider is called to provide its items.
+When registering a [UiItemsProvider]($appui-react) with the [UiItemsManager]($appui-react) it is possible to pass an additional argument to limit when the provider is allowed to provide its items. The interface [UiItemProviderOverrides]($appui-react) defines the parameters that can be used to limit when the provider is called to provide its items.
```

Comment:
i didnt realize UiItemsManager static didnt move into a property of UiFramework.
not much we can do now, just mentioning